ams/api/src/main/java/com/netease/arctic/ams/api/properties/CatalogMetaProperties.java

```diff
@@ -25,17 +25,31 @@ public class CatalogMetaProperties {
   public static final String STORAGE_CONFIGS_KEY_HDFS_SITE = "hadoop.hdfs.site";
   public static final String STORAGE_CONFIGS_KEY_CORE_SITE = "hadoop.core.site";
   public static final String STORAGE_CONFIGS_KEY_HIVE_SITE = "hive.site";
+  public static final String STORAGE_CONFIGS_KEY_REGION = "storage.s3.region";
+  public static final String STORAGE_CONFIGS_KEY_ENDPOINT = "storage.s3.endpoint";
+  public static final String STORAGE_CONFIGS_KEY_S3_REGION = "client.region";
+  public static final String STORAGE_CONFIGS_KEY_S3_ENDPOINT = "s3.endpoint";
 
+  @Deprecated
```

Comment:
This property is not easy to be removed, considering to be compatible with history catalog configurations. 

I think it may be more appropriate to remove the `@Deprecated` and rename the variable to `STORAGE_CONFIGS_VALUE_TYPE_HDFS_LEGACY`.